test/test_other.py

```diff
@@ -11484,23 +11484,36 @@ def test_syslog(self):
     self.do_other_test('test_syslog.c')
 
   @parameterized({
-    '': (False,),
-    'custom': (True,),
+    '': (False, False),
+    'custom': (True, False),
+    'jspi': (False, True),
```

Comment:
Would it be possible to support the custom loader and JSPI at the same time? The custom loader is meant to allow eager initialization of the secondary module, so it would be useful even with JSPI.

Reply:
It should be possible. I think we need to do some more work to wire it up though.